.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
this may seem odd, but it's a standard husky thing, created during the 'prepare' phase of the package.json where husky installs itself.

Reply:
Is this documented by husky? 🤔

Reply:
it's documented via comment in the directory entry / file inside the directory entry. I admit I find it odd